js/parse.js

```diff
@@ -3,50 +3,21 @@
   Group: Team Stimulus
   Created At: 2/28/15
 
-  This handles the event bindings for the input form and the code to parse the
-  ratings.csv file that is imported.
+  This Parse module handles the code to parse the
+  ratings.csv file that is imported into an array of pictures.
 
   All code in this file was authored by David Lin
 */
 
-$(function() {
-  var directory = new air.File();
-  var picturePath = "";
-  var ratingsFile;
-  var pictures = [];
-
-  /*
-    When the #picture-directory input is clicked, use AIR's browseForDirectory() method
-    When the directory is selected, save the path and display it in the form.
-  */
-  $('#picture-directory').click(function(e) {
-    e.preventDefault();
-    directory.addEventListener(air.Event.SELECT, function(e) {
-      picturePath = directory.nativePath;
-      $('#picture-directory').text(picturePath);
-    });
-    directory.browseForDirectory("Choose the picture directory");
-  });
+var Parse = (function() {
+  var my = {};
 
-  /*
-    When submit is clicked on the form, parse all of the input into variables
-    and call parse() to parse the ratings file.
-  */
-  $('#input-form').submit(function(e) {
-    e.preventDefault();
-    air.trace("parsing: " + $('#ratings-file').val());
-    ratingsFile = new air.File($('#ratings-file').val());
-    numGroups = $('#num-groups').val();
-    picsPerGroup = $('#pics-per-group').val();
-    avgRating = $('#rating-per-group').val();
-    air.trace("numGroups: " + numGroups + ", picsPerGroup: " + picsPerGroup + ", avgRating: " + avgRating);
-    parse();
-  });
+  var pictures = [];
```

Comment:
Is this module-scoped var necessary? Doesn't seem to be used anywhere else, can be locally-scoped in my.getPictures(...)